support-frontend/assets/helpers/subscriptionsForms/submit.ts

```diff
@@ -225,7 +225,7 @@ function buildRegularPaymentRequest(
 		csrUsername,
 		salesforceCaseId,
 		debugInfo: actionHistory,
-		threeTierCreateSupporterPlusSubscription: inThreeTierV2Variant(
+		threeTierCreateSupporterPlusSubscription: showThreeTierCheckout(
```

Comment:
We're calling `showThreeTierCheckout(state.common.abParticipations)` in 2 places in this module, both here and when we assign it to the variable `inThreeTier`, could we not use `inThreeTier` in  this case to save calling it multiple times? It may mean moving the declaration of `inThreeTier` so it's accessible here.

Reply:
Yes much neater, moved out declaration to single place, updated

Reply:
Yes this is neater, `inThreeTier` declaration moved up, updated